helm/ffc-pact-broker/values.yaml

```diff
@@ -12,7 +12,8 @@ container:
   limitMemory: 120Mi
   limitCpu: 120m
   readOnlyRootFilesystem: true
-  allowPrivilegeEscalation: false  
+  allowPrivilegeEscalation: false
+  webhookHostWhitelist: ffc-jenkins.aws-int.defra.cloud ffc-notifications.slack.com github.com
```

Comment:
Is the ffc-jenkins url a 'secret'?

Reply:
i.e. Are we ok to publish it to github?

Reply:
No we can't have this in GitHub.  Although our current helm workflow is "clunky" we should still follow it.  So this value should be injected through a `dev-values.yaml` during deployment.

In future it will be mapped from a config store to a config map, but it won't be in the public repo.